scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/rmproxies/RMProxy.java

```diff
@@ -60,6 +61,8 @@
 
     private Credentials creds;
 
+    private Map<String, HybridEncryptionUtil.HybridEncryptedData> thirdPartyCredentials;
+
     RMProxy(URI rmURL, Credentials creds) throws RMException, RMProxyCreationException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)